### PR TITLE
Fixing problem with visibility of floating elements

### PR DESCRIPTION
**Pull request by Ksz-K**
As in subject.
Duo to swipe functionalities the border shadow that has been expected to hide flying in elements to carousel does not work properly. The root of the problem was  z-index applied to carousel inner DIV.
Negative makes border shadow working properly however at the same time kills swipe functionalities.
It was the reason to remove  border shadow and add inner DIV with position absolute that changes its size dynamically and disappears at 1200+ resolution.

### CODE REVIEW

File: src/partials/80_section-brands-carousel.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
-<div class="container overflow-hidden right-shadow py-3 px-6 mt-2 mb-1 border">
+<div class="container py-3 px-6 mt-2 mb-1 border-top border-bottom">
   <div id="carouselControls" class="carousel slide " data-ride="carousel">
-    <div class="carousel-inner">
-      <div class="carousel-item active">
+    <div class="carousel-inner" style="position: relative;">
```
Comment (Neonides) [inline src/partials/80_section-brands-carousel.html]: inline style? Mówimy nie takim rzeczom

File: src/partials/80_section-brands-carousel.html [hunk 2]
```diff
@@ -32,7 +32,7 @@
           alt="logo"
         />
       </div>
-      <div class="carousel-item">
+      <div class="carousel-item ">
```
Comment (Neonides) [inline src/partials/80_section-brands-carousel.html]: Tutaj spacja się wkradła

File: src/partials/80_section-brands-carousel.html [hunk 3]
```diff
@@ -1,7 +1,7 @@
-<div class="container overflow-hidden right-shadow py-3 px-6 mt-2 mb-1 border">
+<div class="container py-3 px-6 mt-2 mb-1 border-top border-bottom">
   <div id="carouselControls" class="carousel slide " data-ride="carousel">
-    <div class="carousel-inner">
-      <div class="carousel-item active">
+    <div class="carousel-inner" style="position: relative;">
+      <div class="carousel-item  active">
```
Comment (Neonides) [inline src/partials/80_section-brands-carousel.html]: dwie spacje

File: src/sass/components/_brands-carousel.scss [hunk 4]
```diff
@@ -62,17 +62,11 @@
...
 
-.container.right-shadow {
-  box-shadow: 50px 0px 0px rgba(255, 255, 255, 1);
+.hide-flowing-imgs {
+  position: absolute;
+  right: 0;
+  top: 0;
+  width: 7%;
+  height: 12vh;
+  background-color: white;
```
Comment (Neonides) [inline src/sass/components/_brands-carousel.scss]: Kolor mamy w zmiennych

**Comment by Ksz-K** [comment]
Code amendment done.

**Pull request merged**